### DISCUSSION TITLE
Livewire v3.01 ComponentNotFound Exeption

### DISCUSSION THREAD

**Discussion by rretzko** [livewire/livewire#6816] [1. Help] [answered]
Hi - I'm hitting a ComponentNotFound Exception on a new Laravel 10/Livewire3 instance deployed on AWS Lamba via Vapor.  The log reads: 

{
"userId": 2626,
"exception": 
{
"class": "Livewire\Exceptions\ComponentNotFoundException",
"message": "Unable to find component: [schools.test-component]",
"code": 0,
"file": "/var/task/vendor/livewire/livewire/src/Mechanisms/ComponentRegistry.php:126"
},
"aws_request_id": "93b692da-9bf2-463e-b940-b4cb2bc0c2be"
}
I do NOT hit this error on my local machine (Windows), where Livewire 3 is working great! 
I first thought it might be a file/directory naming issue between Windows and *nix, so I also tried implementing the Counter example from the docs and hit the same exception.
I'd appreciate some help troubleshooting the problem as it is occurring wherever I'm using Livewire 3.
I've attached screenshots of:
- Directory structure
- TestComponent object
- test-component view
![TestComponent_Object](https://github.com/livewire/livewire/assets/16844486/39d4a7b7-d475-498c-b213-a28091b5d14f)
![text-component_View](https://github.com/livewire/livewire/assets/16844486/a056f9ad-b024-46a4-a81d-68c402e71be6)
![directoryStructure](https://github.com/livewire/livewire/assets/16844486/397c0b19-d52c-4df5-a7fe-e106c14b82c3)
Any help in troubleshooting this is appreciated!

**Comment by joshhanley** [comment, marked as answer]
@rretzko unix is case sensitive, try renaming your `app/livewire/schools` to `app/Livewire/Schools` and see if that works.

**Reply by rretzko** [reply]
Thanks, Josh - This fixed it.  I thought I had that sussed out, but obviously not quite.
Your turnaround time is especially appreciated!
